api/validation/swagger_api_validation.yml

```diff
@@ -30,7 +30,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/User'
-    put:
+    post:
```

Comment:
The summary tells "Updates user data information"... We should either use put method with verb update in summary or use post method with verb create in summary

Reply:
Well, we do a POST so I changed the verb in the summary